Src/Sentry.Xamarin.Forms/Internals/SentryXamarinFormsIntegration.cs

```diff
@@ -1,25 +1,19 @@
 ﻿using Sentry.Extensions;
-using Sentry.Xamarin.Forms.Extensions;
-using Sentry.Xamarin.Internals;
+using Sentry.Integrations;
 using System.Collections.Generic;
-using System.Threading.Tasks;
 using Xamarin.Forms;
 using Xamarin.Forms.Internals;
 
 namespace Sentry.Xamarin.Forms.Internals
 {
-    internal class SentryXamarinFormsIntegration : IPageNavigationTracker
+    internal class SentryXamarinFormsIntegration : ISdkIntegration
     {
-        internal static SentryXamarinFormsIntegration Instance;
-        private SentryXamarinOptions _options;
-        private DelegateLogListener _xamarinLogger;
-        private IHub _hub;
+        internal static SentryXamarinFormsIntegration Instance { get; set; }
+        private SentryXamarinOptions _options { get; }
+        private DelegateLogListener _xamarinLogger { get; set; }
+        private IHub _hub { get; set; }
```

Comment:
I wouldn't make these properties if they're remaining private, unless there's a good reason.

Rule of thumb is: public properties, private fields.  `readonly` fields are even better when possible.